### PR TITLE
Update README.md for `distributed_debate`

### PR DESCRIPTION
**Pull request by zyzhang1130**
reformated README file and updated tested models

---
name: Pull Request
about: Create a pull request
---
## Description

[Please describe the background, purpose, changes made, and how to test this PR]

## Checklist

Please check the following items before code is ready to be reviewed.

- [ ]  Code has passed all tests
- [ ]  Docstrings have been added/updated in Google Style
- [x]  Documentation has been updated
- [ ]  Code is ready for review

### CODE REVIEW

File: examples/distributed_debate/README.md [hunk 1]
```diff
@@ -1,54 +1,74 @@
...
+
+These models are tested in this example. For other models, some modifications may be needed.
+- Ollama Chat (qwen2:1.5b)
+- Dashscope Chat (qwen-Max)
+- Gemini Chat (gemini-pro)
+
+## Prerequisites
+
+Before running the example:
+- Install the distributed version of AgentScope
```
Comment (ZiTao-Li) [inline examples/distributed_debate/README.md:28]: Provide the exact command here?

File: examples/distributed_debate/README.md [hunk 2]
```diff
@@ -1,54 +1,74 @@
...
+For an LLM-based Pro:
 ```shell
 cd examples/distributed_debate
-# setup LLM-based Pro
 python distributed_debate.py --role pro --pro-host localhost --pro-port 12011
-# or join the debate as Pro by yourself
-# python distributed_debate.py --role pro --pro-host localhost --pro-port 12011 --is-human
 ```
 
+For human participation as Pro:
```
Comment (ZiTao-Li) [inline examples/distributed_debate/README.md:43]: Indicate that it is a alternative to the LLM-base, not sequential

**Review by ZiTao-Li — COMMENTED** [review]
see inline comments

File: examples/distributed_debate/README.md (hunk 2 above)
Comment (zyzhang1130) [reply examples/distributed_debate/README.md]: Done.

**Review by pan-x-c — APPROVED** [review]
LGTM

**Pull request merged**